src/main/java/fr/esgi/al5_2/Tayarim/controllers/ProprietaireController.java

```diff
@@ -26,140 +24,147 @@
-        ex.getBindingResult().getAllErrors().forEach((error) -> {
-            String errorMessage = error.getDefaultMessage();
-            errors.add(errorMessage);
-        });
-
-        errorMapping.put("errors", errors);
-
-        return errorMapping;
-    }
-
-    @ResponseStatus(HttpStatus.BAD_REQUEST)
-    @ExceptionHandler(ProprietaireInvalidUpdateBody.class)
-    public Map<String, List<String>> proprietaireInvalidUpdateBody(ProprietaireInvalidUpdateBody ex){
-        return mapException(ex);
-    }
-
-    @ResponseStatus(HttpStatus.CONFLICT)
-    @ExceptionHandler({ProprietaireEmailAlreadyExistException.class})
-    public Map<String, List<String>> proprietaireEmailAlreadyExistException(ProprietaireEmailAlreadyExistException ex){
-        return mapException(ex);
-    }
-
-    @ResponseStatus(HttpStatus.CONFLICT)
-    @ExceptionHandler({ProprietaireNumTelAlreadyExistException.class})
-    public Map<String, List<String>> proprietaireNumTelAlreadyExistException(ProprietaireNumTelAlreadyExistException ex){
-        return mapException(ex);
-    }
-
-    @ResponseStatus(HttpStatus.NOT_FOUND)
-    @ExceptionHandler({ProprietaireNotFoundException.class})
-    public Map<String, List<String>> proprietaireNotFoundException(ProprietaireNotFoundException ex){
-        return mapException(ex);
-    }
-
-    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
-    @ExceptionHandler({PasswordHashNotPossibleException.class})
-    public Map<String, List<String>> passwordHashNotPossibleException(PasswordHashNotPossibleException ex){
-        return mapException(ex);
-    }
-
-    @ResponseStatus(HttpStatus.UNAUTHORIZED)
-    @ExceptionHandler({TokenExpireOrInvalidException.class})
-    public Map<String, List<String>> tokenExpireOrInvalidException(TokenExpireOrInvalidException ex){
-        return mapException(ex);
-    }
-
-    @ResponseStatus(HttpStatus.UNAUTHORIZED)
-    @ExceptionHandler({UnauthorizedException.class})
-    public Map<String, List<String>> unauthorizedException(UnauthorizedException ex){
-        return mapException(ex);
-    }
-
-    private Map<String, List<String>> mapException(RuntimeException exception){
-        ArrayList<String> errors = new ArrayList<>();
-        Map<String, List<String>> errorMapping = new HashMap<>();
-        errors.add(exception.getMessage());
-
-        errorMapping.put("errors", errors);
-
-        return errorMapping;
-    }
-}
+  private final ProprietaireService proprietaireService;
+  private final AuthService authService;
+
+  public ProprietaireController(ProprietaireService proprietaireService, AuthService authService) {
+    this.proprietaireService = proprietaireService;
+    this.authService = authService;
+  }
+
+  @PostMapping("")
+  public ResponseEntity<ProprietaireDTO> creerProprietaire(
+      @RequestHeader("Authorization") String authHeader,
+      @Valid @RequestBody ProprietaireCreationDTO proprietaireCreationDTO) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireCreationDTO' must contain no more than '1' consecutive capital letters.